ConsoleApp2/ConsoleApp2/SOLID/SingleResponsibilityPrinciple.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace LearnBasics.Sandbox.SOLID
+{
+    class SingleResponsibilityPrinciple
+    {
+        public void Example()
+        {
+            var store = new MobileStore(
+                new ConsolePhoneReader(), new GeneralPhoneBinder(),
+                new GeneralPhoneValidator(), new TextPhoneSaver());
+            store.Process();
+        }
+    }
+    class Phone
+    {
+        public string Model { get; set; }
+        public int Price { get; set; }
+    }
+
+    class MobileStore
+    {
+        List<Phone> phones = new List<Phone>();
```

Comment:
please name private fields with underscore at the beginning (_)

Reply:
done